learnupon_exporter/management/commands/learnupon_export_enrollment_data.py

```diff
@@ -113,9 +113,9 @@ def export_enrollment_data(self, csv_file):
                 row["Enrollment Status"] = "completed"
                 row["Enrollment Started Date"] = first_block_viewed_at
 
-            if grade:
+            if grade and grade.passed_timestamp:
                 row["Enrollment Score"] = grade.percent_grade
```

Comment:
The client said they don't care about score -- could you remove this line?

Reply:
Sure. With the other changes it means that I don't need to use the `PersistentCourseGrade` anymore since `first_block_viewed_at` will determine completion.